Analysis/backgrounds_zgwg.tex

```diff
@@ -199,6 +204,7 @@ \subsection{Data-driven Control Regions}
 Figure~\ref{fig:tf_w} shows the transfer factor \RWe\ (\RWm) between the single-electron (single-muon) control region and the combined signal regions, for which the numerator is the estimated \wlng\ yield in the combined signal regions, and the denominator is the estimated \wlng\ yield in the relevant control region. 
 The ratio \RWl\ decreases with increasing \ETg\ in a similar manner to \RZll.
 The underlying logic is the same; e.g., that the signal region selection efficiency is unaffected by \ETg\ while the control region acceptances increase with increasing \ETg\ due to increased lepton efficiency resulting from lower \PW\ rapidity. 
+\PH{the logi is there, but this seems a little wonky. Its really the likelihood of a lepton to be out of acceptance varies with pT, so the failure rate grows with low pT.}
```

Comment:
I feel this is two different ways of saying the same thing. lower rapidity = higher pt